README.md

```diff
@@ -66,30 +102,55 @@ Welcome to the **SimplyLang** language project! SimplyLang is a programming lang
 
    Save the file as `example.SimplyLang` and run it using the command above to see the result.
 
-## Project Goals
+## 📂 Project Structure:
```

Comment:
this resource was generated only recently, but is there any chance you could include this in the project information section? 
https://deepwiki.com/AffanShaikhsurab/SimplyLang 
would be a nice addition.